docs/unified-test-documentation/dasharo-security/206-secure-boot.md

```diff
@@ -499,3 +499,914 @@ incorrect format.
 **Expected result**
 
 The popup with information about `ERROR: Unsupported file type!` should appear.
+
+## SBO009.001 Attempt to boot file signed for intermediate certificate
+
+**Test description**
+
+This test verifies that a file signed with an intermediate certificate can be
+executed.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo.
+1. Additional `USB storage` - at least 1GB - for keeping files for booting.
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Run script `open-source-firmware-validation/scripts/secure-boot/generate-images/sb-img/wrapper.sh`
+    to generate `INTERMIDIATE.img`.
+1. Place `open-source-firmware-validation/scripts/secure-boot/images/INTERMIDIATE.img`
+    on the `USB storage`.
+1. Plug the `USB storage` into DUT.
+1. Power on the DUT.
+1. While the DUT is booting, hold the `BIOS_SETUP_KEY` to enter the UEFI setup
+    menu.
+1. Enter the `Device Manager` menu using the arrow keys and Enter.
+1. Enter the `Secure Boot Configuration` submenu.
+1. Set the `Current Secure Boot State` field to `Enabled`.
+1. Set the `Secure Boot Mode` field to `Custom Mode`.
+1. Select options in the given order: `Custom Secure Boot Options` ->
+    `DB Options` -> `Enroll Signature` -> `Enroll Signature Using File`.
+1. Select the certificate from the `USB storage`.
+1. Select the `Commit Changes and Exit` option.
+1. Press `ESC` until the setup menu.
+1. Select the `Reset` option.
+1. While the DUT is booting, hold the `BOOT_MENU_KEY` to enter the boot menu.
+1. Select the `UEFI Shell` option using the arrow keys and press `Enter`.
+1. In the shell open the `USB storage` by executing the following command:
+
+    ```bash
+    FS0:
+    ```
+
+    > One of the filesystems in the FS list will be the USB storage - typically
+    > `FS0:`
+
+1. Boot the previously prepared file by typing its full name:
+
+    ```bash
+    signed-hello.efi
+    ```
+
+**Expected result**
+
+1. File boots correctly (no information: `Command Error Status: Access Denied`
+    on the output).
+1. The output of the command shows file content.
+
+Example output:
+
+```bash
+Hello, world!
+```
+
+## SBO010.001 Check support for rsa2k signed certificates
+
+**Test description**
+
+This test verifies that a file can be booted via Secure Boot using an RSA2048
+signed certificate.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo.
+1. Additional `USB storage` - at least 1GB - for keeping files for booting.
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Run script `open-source-firmware-validation/scripts/secure-boot/generate-images/sb-img-wrapper.sh`
```

Comment:
@artur-rs I was looking for a better solution in tests `SB0010` and `sb0011` but did not find one. `pre-commit` argues about these lines because it exceeds 80 characters.

Reply:
@DaniilKl 80-char wrap check is omitting the code blocks, the problem was here: bd029699d789edca6e0d949cca9a30d6de16177d